service/src/test/java/org/ehrbase/aql/compiler/InvokeVisitorTest.java

```diff
@@ -30,121 +30,22 @@
 
 public class InvokeVisitorTest {
```

Comment:
I would suggest to implement a standalone unit test that doesn't require an external service. F.e. using MockServer:

- https://www.mock-server.com/
- https://www.baeldung.com/mockserver

This would allow to enrich the test suite as required (authentication, other query types or formats etc.)

Reply:
yes. we have this in netx sprint task as part of the exception management. I will develop the server mockup and mock exceptions as well.